src/pages/api/firstTimeUserAndEntries.ts

```diff
@@ -0,0 +1,48 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getSupabase } from '../../utils/supabase';
+import { getAuthenticatedUserFromSession } from '@/utils/passage';
+import { getZodiacSign } from '@/utils/utils';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse): Promise<void> {
+	const { name, birth_date, last_cycle_start, last_cycle_length } = req.body;
```

Comment:
```suggestion
	const { name, birth_date, last_cycle_start, last_cycle_length }: FirstTimeUser = req.body;
```

If TypeScript knows this data type "is the same thing" as the data type being sent by the `apiCalls` util, TypeScript can be more helpful: preventing typos, preventing client + server from getting out of sync (if a property is changed or removed), and so on 👍